tests/unit/UpgradeContainer/FilesystemAdapterTest.php

```diff
@@ -286,7 +286,7 @@ public function testTempFolderIsNotAPrestashopReleaseAfterChanges()
 
     protected function fillFolderWithPsAssets($folder)
     {
-        mkdir($folder);
+        mkdir($folder, 0777, true);
```

Comment:
why we have to do this ?

Reply:
The test was failing for some reason. Let me revert this.